tests/unit/amo/components/TestAddonReviewManager.js

```diff
@@ -69,15 +71,23 @@ describe(__filename, () => {
   });
 
   it('renders a DismissibleTextForm formFooter', () => {
+    const linkParts = getLocalizedTextWithLinkParts({
+      i18n: fakeI18n(),
+      text: 'Please follow our %(linkStart)sreview guidelines%(linkEnd)s.',
+    });
+
     const root = render();
 
     const form = root.find(DismissibleTextForm);
     expect(form).toHaveProp('formFooter');
 
-    const formFooter = staticRender(form.prop('formFooter'));
-    expect(formFooter.text()).toContain('Please follow our review guidelines');
+    const formFooter = shallow(form.prop('formFooter'));
+
+    expect(formFooter.childAt(0).text()).toEqual(linkParts.beforeLinkText);
```

Comment:
We're testing `beforeLinkText` and `innerLinkText` but not `afterLinkText`. Should we also be testing the latter?

Reply:
yes I thought that I added that. There is technically no after text here but I did add this to the component with adding this expect in mind. thanks for catching this.